getting-started/configuration.rst

```diff
@@ -56,33 +60,33 @@ the release as part of the Basic Example:
      <Module file="$(InPath)\BasicExampleLibrary.dll" />
    </Obfuscator>
 
-In the example configuration, two variables are defined, InPath and OutPath,
-using the Var element, and two assemblies are listed for obfuscation, an
+In this sample, the two variables InPath and Output are defined
+using the ``Var`` element. The sample also specifies that two assemblies should be obfuscated: an
 executable and a dll.
 
-Variables defined using the Var element will be expanded in strings following
-the definition...After defining InPath as follows:
+Variables defined using the ``Var`` element will be expanded in strings following
+the definition. For example, when InPath is defined as:
 
 .. code-block:: xml
 
    <Var name="InPath" value=".\Obfuscator_Input" />
 
-It can be used in another location:
+it can be used in a module:
 
 .. code-block:: xml
 
    <Module file="$(InPath)\BasicExampleExe.exe" />
 
-In addition to being usable like macros, there are a few special variables
-that have additional effects. The variable InPath is used when trying to find
-dependencies (the specified path is searched), and the variable OutPath is
-used as the output path for the obfuscated assemblies and the map. If either
-InPath or OutPath is unspecified, they default to the current path (".").
+A few special variables have functionality besides supporting variable expansion:
+
+- The variable ``InPath`` is used when resolving dependencies by searching the specified path. The default of ``Inpath`` is the current working directory (".").
+- The variable OutPath is used as the output path for the obfuscated assemblies and the logfile specified in the variable ``LogFile``. 
+ The default of ``Inpath`` is the current working directory (".").
```

Comment:
I don't think the changes of this paragraph cover the original key points. Also "The default of Inpath" appears twice now.

Reply:
Tried to reformulate. Unsure what you mean by not convering original key points. Can you help me what you mean?